src/networks.json

```diff
@@ -1788,6 +1788,26 @@
     "start": 7521509,
     "logo": "ipfs://QmNnGPr1CNvj12SSGzKARtUHv9FyEfE5nES73U4vBWQSJL"
   },
+  "1733320111": {
+    "key": "1733320111",
+    "name": "Nusphere DevNet",
+    "chainId": 1733320111,
+    "network": "Nusphere DevNet",
+    "testnet": true,
+    "multicall": "0x7b55399d0b394b32b6f1974892be3f8d9e659475",
+    "rpc": [
+      {
+        "url": "https://u0p4xgo08c-u0mrdoctj2-rpc.us0-aws.kaleido.io/",
+        "user": "u0bzotd8yx",
+        "password": "610jGOOaAMlgzFRic40_QueqbJolGulwo0r9JiuE9VQ"
+      }
+    ],
+    "explorer": {
+      "url": "https://nusphere-testnet.tryethernal.com"
+    },
+    "start": 166736,
+    "logo": "ipfs://QmVAYofHEKwb6GjWuoZutT85SbhkgeSNcNk7BohHawnkcd"
+  },
```

Comment:
Hi @aniketpatil8824 I can't find the network here https://chainlist.org/?search=1733320111&testnets=true?? can I refer to the official documentation of this network?

Reply:
Hi @ChaituVR, the chain ID refers to the Kaleido network. We can consider Kaleido as a Web-3 platform for digital assets and acts as a permissioned Blockchain platform. tryethernal is the public explorer of the Kaleido platform. Both Kaleido and tryethernal are in sync in terms of Block number, transactions, contracts etc. 
Below are some useful links that you can refer for the network.
1. [kaleido docs](https://docs.kaleido.io/)
2. [Kaleido Advanced Blockchain configuration](https://docs.kaleido.io/using-kaleido/blockchain-advanced-configuration/#chain-id)
3. [API reference - create an environment for the Kaleido platform's network/consortia](https://api.kaleido.io/platform.html?_gl=1*16bzzus*_ga*ODkyNTA3NzIxLjE2NjAxMjU3Mjg.*_ga_MDZYXC6S7G*MTY4MzA5NDU1Ny40NTUuMS4xNjgzMDk0NjUwLjQyLjAuMA..#tag/Environments/paths/~1consortia~1%7Bconsortia_id%7D~1environments/get)
4. [Ethernal documentation](https://doc.tryethernal.com/)

Reply:
A screenshot of the Kaleido network reflecting chain ID is attached below for your reference. 

<img width="946" alt="kaleido chainID" src="https://user-images.githubusercontent.com/95903132/235849082-b00c4f8d-e47b-47b6-bdb8-8c027f346271.PNG">

Reply:
thanks @aniketpatil8824 is there a way to verify this contract code? https://nusphere-testnet.tryethernal.com/contract/0x7b55399d0b394b32b6f1974892be3f8d9e659475?tab=code 

Everything else looks good :)

Reply:
I still can't see it @aniketpatil8824 Can share me the exact link?